### PR TITLE
Adding option to exclude hadoop and hive jars from gobblin-dist.tar.gz

### PR DESCRIPTION
**Pull request by sahilTakiar**
This will help with the following bug that a few people on the google-groups have hit: https://github.com/linkedin/gobblin/wiki/FAQs#how-do-i-fix-unsupportedfilesystemexception-no-abstractfilesystem-for-scheme-null

Also, created this wiki to document all Gradle build options: https://github.com/linkedin/gobblin/wiki/Gobblin-Build-Options

@chavdar can you review?

### CODE REVIEW

File: gobblin-distribution/build.gradle [hunk 1]
```diff
@@ -17,6 +17,17 @@ repositories {
   }
 }
 
+configurations {
+  compile {
+    if (project.hasProperty('excludeHadoopDeps')) {
```
Comment (chavdar) [inline gobblin-distribution/build.gradle]: I suggest that you also check rootProject.hasProperty('excludeHadoopDeps') if it has been specified at the root level.

**Comment by chavdar** [comment]
Please also update README.md to document the properties.

**Comment by sahilTakiar** [comment]
@chavdar updated. Comments addressed. Updated the `README.md` file include a link to the new wiki page documenting Gobblin Gradle Build Options (https://github.com/linkedin/gobblin/wiki/Gobblin-Build-Options).

**Comment by sahilTakiar** [comment]
@chavdar any other comments?

**Comment by chavdar** [comment]
LGTM

**Pull request merged**